### DISCUSSION TITLE
Best UK map?

### DISCUSSION THREAD

**Discussion by Cackhandcode** [joshuafuller/ATAK-Maps#21] [General]
Hi all,
Would anyone know which maps are most useful in the UK?, I find that all the ATAK default ones have no detail.
Thanks

**Comment by juscghwe** [comment]
I (GER) mostly find myself using Bing, Google and ESRI. Skipping through all those three helps me understanding the area better because they most of the time come from different times in the year. OCM also turned out to be helpful quite some times, especially when operating in dense woods where Google etc reach their limits regarding trails.

**Comment by alsaqqr** [comment]
Hello, my dear friends. Is it possible to add this feature( Nga most current) to the ATAK application?
